src/main/java/org/zalando/nakadi/security/ClientResolver.java

```diff
@@ -27,11 +28,14 @@
 
     private final SecuritySettings settings;
     private final FeatureToggleService featureToggleService;
+    private final String defaultId;
 
     @Autowired
-    public ClientResolver(final SecuritySettings settings, final FeatureToggleService featureToggleService) {
+    public ClientResolver(final SecuritySettings settings, final FeatureToggleService featureToggleService,
```

Comment:
I've refactored this class because I need to now clientId to throttle. By default if AuthMode is OFF we are using adminClientId.